acceptance-setup/package.json

```diff
@@ -0,0 +1,10 @@
+{
+  "name": "acceptance-setup",
+  "private": true,
+  "version": "0.0.1",
+  "devDependencies": {
+    "fs-extra": "5.0.0",
+    "request": "^2.83.0",
```

Comment:
where are you using request?

Reply:
It goes with https://github.com/opencomponents/base-templates/pull/255/files/90d8461105b58191d8e5af76863c740c2cae28ac#diff-c4ef5516c76bde2f4d022b4ef85efb6cR8

as a follow-up we could move to a promise-based request library that doesn't need this or simply promisify it.